packages/postcss-reduce-idents/src/lib/grid-template.js

```diff
@@ -53,7 +58,10 @@ module.exports = function () {
         declCache.push(node);
       } else if (gridChildProperties.has(node.prop.toLowerCase())) {
         valueParser(node.value).walk((child) => {
-          if (child.type === 'word' && !RESERVED_KEYWORDS.has(child.value)) {
+          if (
+            child.type === 'word' &&
+            !RESERVED_KEYWORDS.has(child.value.toLowerCase())
```

Comment:
@jordydhoker in the previous PR, we forgot that the child value could be an uppercase reserved keyword)